src/applications/caregivers/components/FormFields/AddressWithAutofill.jsx

```diff
@@ -78,10 +74,9 @@ const PrimaryAddressWithAutofill = props => {
       // uncheck autofill since we have modified the input value
       if (formData['view:autofill']) formData['view:autofill'] = false;
       // send updated date to the form
-      addDirtyField(fieldName);
```

Comment:
Removing marking a field dirty on input makes the showError validation below more natural. We will only make a field dirty on blur to allow for the user to complete any initial inputs before validation kicks in.